### PR TITLE
add list resolvers to OmegaConf to get items from list

### PR DESCRIPTION
**Pull request by ofrimasad**

### CODE REVIEW

**Comment by BloodAxe** [comment]
Maybe use `getitem` instead of `list_get` to be consistent with python's naming scheme for getting a key from container?

File: src/super_gradients/common/environment/env_helpers.py [hunk 1]
```diff
@@ -96,6 +96,9 @@ def register_hydra_resolvers():
     OmegaConf.register_new_resolver("class", lambda *args: get_cls(*args), replace=True)
     OmegaConf.register_new_resolver("add", lambda *args: sum(args), replace=True)
     OmegaConf.register_new_resolver("cond", lambda boolean, x, y: x if boolean else y, replace=True)
+    OmegaConf.register_new_resolver("getitem", lambda lst, i: lst[i], replace=True)                    # get the i'th object from a list (0 based)
```
Comment (shaydeci) [inline src/super_gradients/common/environment/env_helpers.py:99]: I agree with @BloodAxe, lets rename it to list_get.

**Review by shaydeci — CHANGES_REQUESTED** [review]
Just small renaming comment, then LGTM.

**Review by shaydeci — CHANGES_REQUESTED** [review]
see inline comment regarding the docs

File: src/super_gradients/common/environment/env_helpers.py (hunk 1 above)
Comment (ofrimasad) [reply src/super_gradients/common/environment/env_helpers.py]: fixed in [0917439](https://github.com/Deci-AI/super-gradients/pull/441/commits/091743989a9a486ed41d5cbd86556bbb105823b0)

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**